From 71a58758f7a986c82b0613ab960a2be46158a756 Mon Sep 17 00:00:00 2001
From: Elvis Pranskevichus <elvis@magic.io>
Date: Tue, 16 Apr 2019 11:28:23 -0400
Subject: [PATCH 4/4] Add an explicit dependency on edgedb-python

Default to the PyPI version of EdgeDB-Python.  CI and dev can still use
the git version as necessary.
---
 setup.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/setup.py b/setup.py
index 4bb77374..674415cd 100644
--- a/setup.py
+++ b/setup.py
@@ -47,6 +47,8 @@ RUNTIME_DEPS = [
 
     'graphql-core~=2.1.0',
     'promise~=2.2.0',
+
+    'edgedb~=0.6.1',
 ]
 
 CYTHON_DEPENDENCY = 'Cython==0.29.6'
-- 
2.19.2

